web/themes/custom/os2forms_selvbetjening_theme/assets/pdf.css

```
@@ -0,0 +1,136 @@
+@page {
+  margin: 180px 35px 150px 35px;
```

I think we should set the page size to A4:

```suggestion
  size: A4;
  margin: 180px 35px 150px 35px;  
```

Using `px` seems weird for PDF generation. Do we know how long `1 px` actually is (in `mm`, say) when generating a PDF?